### ISSUE TITLE
Console output also to logfile

### ISSUE THREAD

**Issue by bsch63** [PGelss/wave_train#7]
It would be nice to have the console output to be written in parallel to a log-file, for documentation purposes etc. That log-file should have the same filename as the input Python script, but with extension .txt or .log. Hence, in analogy to the image (.png) and animation (.mp4) output that WaveTrain already produces routinely.
